### PR TITLE
feat: continuous releases with pkg.pr.new

### PR DESCRIPTION
**Pull request by Aslemammad**
As promised in https://x.com/asleMammadam/status/1812226230176960720

But now we gotta install the [app](https://github.com/apps/pkg-pr-new) first. And I'd be happy to hear your feedback, Puru! Thank you for the opportunity!

### CODE REVIEW

**Comment by PuruVJ** [comment]
Thanks man! 

~~Ive installed the app. How can I check whether its working?~~ Got it working.

Why does the URL say neotraverse@3?

Running locally gives error: 
CleanShot 2024-07-14 at 14 10 55

File: .github/workflows/cr.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Continuous Releases
+
+on: [push, pull_request]
```
Comment (mmkal) [inline .github/workflows/cr.yml:3]: Just stumbled across this... The recommended setup is on approval for PRs: https://github.com/stackblitz-labs/pkg.pr.new?tab=readme-ov-file#release-approved-pull-requests-only

Comment (Aslemammad) [reply .github/workflows/cr.yml:3]: Yea, exactly, but this approach has no issues at all if fast feedback is more appealing!

**Comment by Aslemammad** [comment]
> Why does the URL say neotraverse@3?

because it is pr number 3 😅
you can also check the check runs on each commit, you'll see the commit version of the package like https://github.com/PuruVJ/neotraverse/runs/27420578949

> Running locally gives error:

shoot, let me see what's the issue

**Comment by Aslemammad** [comment]
I found the issue, what gh gives us in webhooks is the `PuruVJ` name, while the package.json repository field are `puruvj`. That'd mess with the keys stored on pkg.pr.new

now I disabled compact mode, which should fix the issue, meanwhile I'll write in the docs that the package.json repository field should be case sensitive.

**Comment by Aslemammad** [comment]
Thank you for the new commit, but that should be published on npm since we get the trust from there!

**Comment by PuruVJ** [comment]
Still getting this, even with --force

CleanShot 2024-07-14 at 20 33 38

**Comment by Aslemammad** [comment]
Try the new url generated in the comment!

**Comment by PuruVJ** [comment]
npm error is weird. neotraverse/neotraverse

CleanShot 2024-07-14 at 20 34 25

**Comment by PuruVJ** [comment]
Oh now it worked!

**Comment by PuruVJ** [comment]
Cool!

**Comment by Aslemammad** [comment]
Thank you so much for the patience and trials! I appreciate it mate!

**Comment by PuruVJ** [comment]
Do you have anymore changes to do? I will merge when you're done

**Comment by Aslemammad** [comment]
No, everything's done from my side! 🙌

**Comment by PuruVJ** [comment]
Thanks a lot!

**Comment by Aslemammad** [comment]
Your name is mentioned here now 😅

https://github.com/stackblitz-labs/pkg.pr.new/pull/145

**Comment by PuruVJ** [comment]
Whoaaa 🤯😄😅

**Pull request merged**